dist/index.js

```diff
@@ -10134,21 +10154,24 @@ const run = async () => {
 
             const changelogContent = await readFile(`${folder}CHANGELOG.md`, {encoding: 'utf-8'});
             const {
-                isUnreleased, version, date,
+                isUnreleased,
+                version,
+                date,
+                skeleton,
             } = validateChangelog(changelogContent);
 
-            // Checks if the branch is release
-            if (branch === 'release') {
+            // Checks if the branch is release or branches input.
+            if (branches.includes(branch)) {
```

Comment:
Is it all possible branches, or should it be certain group?

Reply:
> Is it all possible branches, or should it be certain group?

Not completely sure, but if for example we have the configuration `branches: ['release', 'hotfix']` if the current branch is one of those branches it will be evaluated.

Maybe the comment should be like this to make it more clear.
```
// Checks if the branch is release or onece of branches input.
```

Reply:
It looks like this is specific branches group, where we want to validate proper release annotations. Shall we maybe call it `release_branches`?

Reply:
sure, I makes it more clear.